src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSTaxInfo.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Helper class for ClassifyReads that stores taxon data
+ */
+public final class PSTaxInfo {
```

Comment:
Came up with PSPathogenReferenceTaxonProperties and added some explanation to the class header comment.